crates/indexer/src/models/stake_models/staking_pool_voter.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+
+use super::stake_utils::StakeResource;
+use crate::{schema::current_staking_pool_voter, util::standardize_address};
+use aptos_api_types::{Transaction as APITransaction, WriteSetChange as APIWriteSetChange};
+use field_count::FieldCount;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+type StakingPoolAddress = String;
+pub type StakingPoolVoterMap = HashMap<StakingPoolAddress, CurrentStakingPoolVoter>;
+
+#[derive(Debug, Deserialize, FieldCount, Identifiable, Insertable, Serialize)]
+#[diesel(primary_key(staking_pool_address))]
+#[diesel(table_name = current_staking_pool_voter)]
+pub struct CurrentStakingPoolVoter {
```

Comment:
Let's might as well keep track of the operator?

Reply:
How?

Reply:
It's part of the StakePool resource